### ISSUE TITLE
Floorp --> Yokoffing applied in Floorp --> can't change some settings later.

### ISSUE THREAD

**Issue by mahcin** [yokoffing/Betterfox#271]
## REPORT
Before applying Yokoffing in Floorp settings I have changed for example:
- "Memory and Performance" via Floorp settings tab to "low performance"
- "Ask to save logins and passwords for web sites" --> tab ON

Now I have clicked to apply all Yokoffing options in Floorp "Customize Floorp with user.js" setting.
By each "Apply" Floorp restarted.

I wanted to verify&tweak after from Floorp settings menu, but not "about:" level.
I checked:
- "Memory and Performance" via Floorp settings, it was "best performance", so Yokoffing worked. I have changed this option back to low performance, restarted, re-checked, and it was correct as set myself, "low performance"
- I went to password option: - "Ask to save logins and passwords for web sites" --> tab was OFF, so Yokoffing worked as should. I changed now "Ask to save logins and passwords for web sites" tab to ON, but couldn't, so I went to about:config to change signon.rememberSignons but I couldn't overwrite it. So, went to user.js and changed there while Floorp wasn't running, switched ON Floorp and it was changed, so browser was able to save passwords again, however, after switching browser OFF and ON again this option was back to original state set by Yokoffing.

**Describe the bug**
So, after applying Yokoffing in Floorp I can change options later regarding performance, but not all regarding security/passwords.

**Expected behavior**
Would be nice to be able manually change these settings. This is my personal computer, no one else has access to it, so nice to have Yokoffing, just one click and most security sorted, ten just switch ON option manually to save passwords in Floorp - which cannot do now.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Floorp
 - Version:  Version 11.7.1 (64-bit) | Firefox: 115.6.0
![y1](https://github.com/yokoffing/Betterfox/assets/155994915/85cb37f9-c20a-4337-afee-0832b9279437)
![y2](https://github.com/yokoffing/Betterfox/assets/155994915/814dbb4d-87e2-40f1-a257-3ff6d651aa73)

**Comment by mahcin** [comment]
To explain more details:
"Yokoffing Securefox" --> "Apply" --> "Ask to save logins and passwords for web sites" --> tab is changed to OFF (as should)
Manually I can change it to ON, it remain ON after restart.

"Yokoffing Default" --> "Apply" --> "Ask to save logins and passwords for web sites" -->  tab is changed to OFF (as should)
Manually I can change it to ON, **it's switched back to OFF after restart.**

**Comment by yokoffing** [comment]
For now, I recommend _not_ using the Floorp provided options. Follow the steps for [Getting Started](https://github.com/yokoffing/Betterfox#getting-started) and use the `user.js`. Then add the override to [enable the password manager](https://github.com/yokoffing/Betterfox/wiki/Common-Overrides#password-credit-card-and-address-management) at the bottom.

Note: Betterfox is for Firefox officially, not Floorp.

As the project grows, I should re-consider moving the password manager setting to [Optional Hardening](https://github.com/yokoffing/Betterfox/wiki/Optional-Hardening) instead of disabling it by default.

**Comment by mahcin** [comment]
Okay, thank you.
I'm aware of fact that Betterfox is for Firefox and not for Floorp. Thank you for growing this project and making it possible unofficially.
Your manual instruction for Firefox does work for Floorp, just tested.

I went a bit further with testing Yokoffing:

1. In Floorp --> "Yokoffing Default" --> Apply, then confirmed "OK" in warning box:

> Applying a new user.js will overwrite your current user.js file.
> Please back up your current user.js file before continuing.

Now in Floorp profile folder new user.js file was created, previously was no user.js file at all in here.
I'm unable to change content in this user.js file, every single time when Floorp is restarted, user.js is re-created with content of "Floorp Default" options and my changes are simply wiped from user.js.
2. Copied in to safe place above user.js which was created by "Yokoffing Default" (untouched).
3. In Floorp --> "Yokoffing Default" --> Apply, then again confirmed "OK" in warning box:

> Applying a new user.js will overwrite your current user.js file.
> Please back up your current user.js file before continuing.

Now in Floorp profile folder user.js file completely disappeared. No user.js file in folder at all, even if Floorp notified:

> Applying a new user.js will overwrite your current user.js file.
> Please back up your current user.js file before continuing.

So at this point settings options must be kept elsewhere then, but by above steps I was able "virtually" copy/paste Yokoffing user.js 
4. Copied back to Floorp Yokoffing user.js from step two, now when Yokoffing is not in use really, but "Floorp Default" in use.
5. Restart
6. Done. Now changes made in Floorp regarding saving passwords still doesn't "save" after Floorp restart, however any new changes in copied Yokoffing user.js (incl above password options) does the job and does make changes to Floorp. I noticed this way Yokoffing user.js don't get overwritten. File keep saved changes after restart, changes are noticeable in Floorp, so here is the way of doing it manually with original Yokoffing file of Floorp. So, by this I understand that there is no problem in Yokoffing user.js itself, but somewhere in Floorp or procedure of adding Yokoffing.
